docs/content/faq.md

```diff
@@ -4,49 +4,91 @@ title: 'FAQ'
 
 ### What kind of database are you using?
 
-None! No database is required. Filesystem is the database.
-This image is based on config files that can be persisted using bind mounts (default) or Docker volumes, and as such versioned, backed up and so forth.
+None! No database is required. The filesystem is the database. This image is based on config files that can be persisted using bind mounts (default) or Docker volumes, and as such versioned, backed up and so forth.
 
 ### Where are emails stored?
 
 Mails are stored in `/var/mail/${domain}/${username}`. Since `v9.0.0` it is possible to add custom `user_attributes` for each accounts to have a different mailbox configuration (See [#1792][github-issue-1792]).
 
-### How to alter the running `docker-mailserver` instance _without_ relaunching the container?
+### How are IMAP mailboxes (_aka IMAP Folders_) set up?
+
+`INBOX` is setup by default with the special IMAP folders `Drafts`, `Sent`, `Junk` and `Trash`. You can learn how to modify or add your own folders (_including additional special folders like `Archive`_) by visiting our docs page [_Customizing IMAP Folders_](https://docker-mailserver.github.io/docker-mailserver/edge/examples/use-cases/imap-folders) for more information.
+
+### How do I update DMS?
+
+**Make sure to read the [CHANGELOG](https://github.com/docker-mailserver/docker-mailserver/blob/master/CHANGELOG.md)** before updating to new versions, to be prepared for possible breaking changes.
+
+Then, run the following commands:
+
+``` BASH
+docker-compose pull
+docker-compose down
+docker-compose up -d mailserver
```

Comment:
Prior `down` command didn't specify a service, if there's more than one, you probably want to bring them back up too? User should know either way.

```suggestion
docker-compose up -d
```